cardano-crypto-class/src/Cardano/Crypto/VRF/Class.hs

```diff
@@ -226,7 +226,7 @@ decodeVerKeyVRF = do
         | actual /= expected
                     -> fail ("decodeVerKeyVRF: wrong length, expected " ++
                              show expected ++ " bytes but got " ++ show actual)
-        | otherwise -> fail "decodeVerKeyVRF: cannot decode key"
+        | otherwise -> fail $ "decodeVerKeyVRF: cannot decode key " <> show bs <> " " <> show (BS.length bs) <> " " <> show actual <> " " <> show expected
```

Comment:
How does changing this error message to include the actual and expected lengths help? We have already checked that the actual and expected length match.

This case is for errors other than length errors. I'm not aware that in practice any instances of this class have any decode errors other than length.

Reply:
Oops, that was for debugging the earlier code that didn't work.  Removing.